### DISCUSSION TITLE
Share and discuss your ideas about LayoutParser!

### DISCUSSION THREAD

**Discussion by lolipopshock** [Layout-Parser/layout-parser#126] [Announcements]
Dear LayoutParser Community, 

LayoutParser is turning three! During the past two years, I am thrilled to witness such great excitement on the tool we've developed, and it feels so great that LayoutParser can be practically helpful for your work. We'll continue to maintain the tool and improve the libraries to better support your document analysis tasks! 

We've been planning a ton of new features and functionalities, but we'd also like to hear from your thoughts and ideas -- Which are the most important functions in your work yet currently missing in the LayoutParser ecosystems? Which part of the library is hard to use? What are the new models that should be included in LayoutParser? Please feel free to share your ideas in Github Discussion, and any suggestions will be greatly appreciated! 

I'd like to thank you for your support for LayoutParser during the past years; and let's work to make it better :)

**Comment by inspire298** [comment]
How to get the dataset?
